src/txdb.h

```diff
@@ -121,6 +121,7 @@ class CBlockTreeDB : public CDBWrapper
     bool ReadLastBlockFile(int &nFile);
     bool WriteReindexing(bool fReindex);
     bool ReadReindexing(bool &fReindex);
+    bool HasTxIndex(const uint256 &txid);
```

Comment:
Again, I understand you are replicating surrounding code, but according to clang, it should be:

```suggestion
    bool HasTxIndex(const uint256& txid);
```